### PR TITLE
Some additions for controllers and JSP files

### PR DESCRIPTION
**Pull request by vakhonin**

### CODE REVIEW

File: src/main/java/com/epam/magazinestore/controller/AccountController.java [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
+        return "login";
+    }
+
+    @RequestMapping(value = "/create_account", method = RequestMethod.POST)
+    public String createAccount(Model model) {
+
+        String message = "Registration successful! Now you can sign in with your login and password:";
+        model.addAttribute("success", message);
+
+        return "login";
```
Comment (Jebsuz) [inline src/main/java/com/epam/magazinestore/controller/AccountController.java]: is it a work in progress ? i don't see any account registration process here

Comment (NiazBikbaev) [reply src/main/java/com/epam/magazinestore/controller/AccountController.java]: In progress

File: src/main/java/com/epam/magazinestore/controller/AccountController.java [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 
 @Controller
 public class AccountController {
 
+
     @RequestMapping(value = "/account", method = RequestMethod.GET)
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/controller/AccountController.java]: RequestMapping to GetMapping, "/account" to PathUtils.class as constant

File: src/main/java/com/epam/magazinestore/controller/AccountController.java [hunk 3]
```diff
@@ -1,16 +1,31 @@
...
     public String account() {
         return "account";
     }
 
+    @RequestMapping(value = "/login", method = RequestMethod.GET)
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/controller/AccountController.java]: GetMapping and "/login" to PathUtils.class

File: src/main/java/com/epam/magazinestore/controller/AccountController.java [hunk 4]
```diff
@@ -1,16 +1,31 @@
...
+    public String login() {
+        return "login";
+    }
+
+    @RequestMapping(value = "/create_account", method = RequestMethod.POST)
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/controller/AccountController.java]: PostMapping and "/create_account" to PathUtils.class

File: src/main/java/com/epam/magazinestore/controller/IndexController.java [hunk 5]
```diff
@@ -1,26 +1,20 @@
...
 
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 
 @Controller
 public class IndexController {
 
+    @RequestMapping(value = "/", method = RequestMethod.GET)
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/controller/IndexController.java]: GetMapping, "/" to PathUtils as constant

File: src/main/java/com/epam/magazinestore/controller/IndexController.java [hunk 6]
```diff
@@ -1,26 +1,20 @@
...
+    public String home() {
+        return "home";
+    }
+
     @RequestMapping(value = "/registration", method = RequestMethod.GET)
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/controller/IndexController.java]: GetMapping and "/registration" to PathUtils as constant

**Pull request merged**